### DISCUSSION TITLE
Compiled at Windows 11

### DISCUSSION THREAD

**Discussion by cressidagp** [arcemu/arcemu#455] [General]
Hi there, just in case than someone its interested, i installed windows 11 on my new "fat lady" and was able to compile and run arcemu.

Used the following software:

- Windows 11 Home 23H2
- CMake 3.28.1
- Visual Studio Professional 2017
- MariaDB 11.4.0 Alpha

I recomend avoid MySQL, tested many versions, the more new drop SSL 1.1 support, some others more old gived query errors at world start, The very old 5.7 wich i used for windows 10 dont work out at windows 11.

pd: its the first time i used MariaDB and im very happy with She :P

**Comment by dfighter1985** [comment]
Well some days ago I posted an image of Windows 11 running Arcemu on Raspberry Pi in the ARM pull request conversation:

![20240120_000037242](https://github.com/arcemu/arcemu/assets/589414/f36dd6ca-ad93-4411-8260-fa1731ed398a)

So it's not new that it works on Windows 11.
However it is news that you could also successfully compile! Thanks for trying!

According to Natrist you can also compile with VS 2022 if that is what you'd like.

As for the SSL support that will have to be checked.
Can you turn off SSL in latest MySQL or is it mandatory?

**Reply by cressidagp** [reply]
You're right! aparently, there is an option SSL_MODE_DISABLED. But probably, then will appear the problem of some querys having sintax error in the Object Manager
